### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#7] [Announcements]
# index

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/

**Comment by sata0109** [comment]
这文章想表达什么？就是随便一篇故事吗？

**Reply by wurencaideli** [reply]
是收集的一些短文，然后每天随机一篇，每天读一篇感觉挺好的

**Reply by sata0109** [reply]
啊我说的是昨天看到有一篇文章，希区柯克的，没看懂他想表达什么哈哈哈哈啊，我知道这个网站是干嘛的。

**Comment by Michael-jim** [comment]
宝藏网站1111

**Comment by fenychn0206** [comment]
### 笑死，必须收藏兄弟们
